db/schema.rb

```diff
@@ -40,5 +40,22 @@
     t.index ["name"], name: "index_publications_on_name"
   end
 
+  create_table "users", force: :cascade do |t|
+    t.string "email", default: "", null: false
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.